README.md

```diff
@@ -105,6 +105,59 @@ steps:
         body: output
       })
 ```
+Specify terraform working directory and previous outputs can be used in subsequent steps for a concise comment on the pull request
+```yaml
+env:
+    tf_actions_working_dir: <Terraform working directory>
+steps:
+- uses: actions/checkout@v2
+- uses: hashicorp/setup-terraform@v1
+
+- name: Terraform fmt
+  id: fmt
+  run: cd ${{ env.tf_actions_working_dir }} && terraform fmt
+  continue-on-error: true
+
+- name: Terraform Init
+  id: init
+  run: cd ${{ env.tf_actions_working_dir }} && terraform init
+
+- name: Terraform Validate
+  id: validate
+  run: cd ${{ env.tf_actions_working_dir }} && terraform validate -no-color
+
+- name: Terraform Plan
+  id: plan
+  run: cd ${{ env.tf_actions_working_dir }} && terraform plan -no-color
+  continue-on-error: true
+
+- uses: actions/github-script@0.9.0
+  if: github.event_name == 'pull_request'
+  env:
+    PLAN: "terraform\n${{ steps.plan.outputs.stdout }}"
+  with:
+    github-token: ${{ secrets.GITHUB_TOKEN }}
+    script: |
+      const output = `#### Terraform Format and Style 🖌\`${{ steps.fmt.outcome }}\`
```

Comment:
I'm not sure where the `outcome` syntax is coming from. Can you clarify?

Reply:
when an `id` is given to step, the status of that step is in JSON and it can be accessed by `${{ steps.<Step Id>.outcome }}` E.g
Below is a sample of a step that ran successfully
```
{
  outputs: {},
  outcome: success,
  conclusion: success
}
```
Below is a sample of how a step that failed but `continue-on-error: true` looks like
```
{
  outputs: {},
  outcome: failure,
  conclusion: success
}
```
As you can see from the above, the step failed but it can be accessed with `${{ steps.<Step Id>.outcome }}`. Since the step was set to continue on error, the `conclusion: success` but the step `outcome: failure`

Reply:
Oh wow. That's pretty hidden in the docs. Found it [here](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#steps-context).